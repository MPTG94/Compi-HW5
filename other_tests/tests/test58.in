void maxTestsTestArithmeticInIf() {
	if (1 < 1 + 1) {
		print("Believe in yourself, take on your challenges, dig deep within yourself to conquer fears. Never let anyone bring you down. You got to keep going. â€“ Chantal Sutherland");
	}
}

void main() {
     maxTestsTestArithmeticInIf();
}
