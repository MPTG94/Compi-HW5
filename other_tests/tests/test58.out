Believe in yourself, take on your challenges, dig deep within yourself to conquer fears. Never let anyone bring you down. You got to keep going. â€“ Chantal Sutherland
